manage

```diff
@@ -37,7 +37,7 @@ validate_apps_gui() { #Given a list of actions and apps, graphically notify the
   for line in $queue ;do
     action="$(echo "$line" | awk -F ';' '{print $1}')"
     
-    if [ "$action" != install ] && [ "$action" != uninstall ] && [ "$action" != update ] && [ "$action" != refresh ];then
+    if [ "$action" != install ] && [ "$action" != uninstall ] && [ "$action" != update ] && [ "$action" != refresh ] && [ "$action" != update-file ];then
```

Comment:
`terminal_manage` should be updated as well even if it is not used currently for `update-file` action (`terminal_manage_multi` is used with your change here https://github.com/Botspot/pi-apps/pull/2340/commits/e12ff09bbc4477b5846f76abbbd43a49e6161a19#diff-196c23750e800642e7c008dc0416536449f295069c8161c23283c86334f8d251R561 )